aiida_firecrest/utils_test.py

```diff
@@ -618,7 +648,32 @@ def task_storage_xfer_external_upload(
         # this skips statuses 111, 112 and 113,
         # see: https://github.com/eth-cscs/pyfirecrest/blob/5edbe85d11b977ed8f6943ca22e4fdc3d6f5e12d/firecrest/BasicClient.py#L143
         # and so we are assuming that the file is uploaded to the server
-
+        # I haven't updated the code to reflect this yet, but the new format of tasks is as follows:
```

Comment:
Please consider to make an issue about this and explain this. I am not sure what this code should do

Reply:
OK this file is now removed. So we don't need this comment anymore.